command/node_status.go

```diff
@@ -384,17 +388,27 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 
 }
 
+func (c *NodeStatusCommand) outputTruncatedNodeDriverInfo(node *api.Node) {
+	c.Ui.Output(c.Colorize().Color("\n[bold]Drivers"))
+	drivers := make([]string, 1)
+
+	for driverName, driverInfo := range node.Drivers {
+		if !driverInfo.Healthy {
+			drivers = append(drivers, fmt.Sprintf("%s (unhealthy)", driverName))
+		} else {
+			drivers = append(drivers, driverName)
```

Comment:
I think we should add a comma to make it clear which driver `(unhealthy)` applies to.